.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: woopra-plugin CI
+
+on: 
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        run: |
+          docker build --build-arg GITHUB_USERNAME=${GITHUB_USERNAME} --build-arg GITHUB_TOKEN=${GITHUB_TOKEN} -t quay.io/${ORGANIZATION}/che-workspace-telemetry-woopra-plugin:${TAG} -f src/main/docker/Dockerfile.multi .
+          echo "::set-env name=IMAGE::quay.io/${ORGANIZATION}/che-workspace-telemetry-woopra-plugin:${TAG}"
+        env:
+          GITHUB_USERNAME: ${{ secrets.GITHUB_USERNAME }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN}}
+          ORGANIZATION: openshiftio
```

Comment:
I think org for upstream should be https://quay.io/organization/eclipse , right?

Reply:
Yes, thank you